### ISSUE TITLE
adding delayed items to an array, time error

### ISSUE THREAD

**Issue by GlauberF** [ReactiveX/rxjs#5445]
## Bug Report

**Current Behavior**
I have an array with the original data and from it, I will add item to item in another array with delay.
But what I notice, when the original item has more data, it seems that the delay time gets longer.

**Reproduction**
```js
private _addItemsArrayAnotherArrayWithDelay(data: Array<any>): void {
        // Init
        from(data)
            .pipe(
                concatMap((item) => of(item).pipe(delay(10))),
                takeUntil(this._unsubscribeAll$)
            )
            .subscribe((el) => {
                // Assign new data with delay.
                this.componentInputDataSource = [
                    ...this.componentInputDataSource,
                    el
                ];
            });
    }
```

**Expected behavior**
it must insert the data informed in the function into the new array (this.componentInputDataSource) in 10ms, regardless of the size of the array informed in the function.
why today it gives the impression that it takes longer when the array is bigger (100 records)

Note: the array informed in the function is of objects, I left it typed as Array <any> for being easier.

**Environment**
- Runtime: node 12.16.3
- RxJS version: 6.5.5
- Angular Cli: 9.1.6

**Comment by cartant** [comment]
> But what I notice, when the original item has more data, it seems that the delay time gets longer.

You are delaying the concatenation of **each array item** by 10 milliseconds, so this seems like the expected behaviour to me.

**Comment by GlauberF** [comment]
@cartant  maybe I expressed myself badly or the role I developed does not meet what I need.

In short, it doesn't matter if the original array passed in the function through the data field, has 10 or 100 positions.

intens should be added to the new array (componentInputDataSource) at the same time.
that is, the 10 initial positions must arrive at the same speed in componentInputDataSource regardless of the size of the original array
